build.gradle

```diff
@@ -317,6 +317,11 @@ task sparkJar(type: ShadowJar) {
     description = "Create a combined jar of project and runtime dependencies that excludes provided spark dependencies"
     configurations = [project.configurations.sparkConfiguration]
     classifier = 'spark'
+
+    //Those have to be excluded to prevent "Invalid signature file digest for Manifest main attributes"
+    exclude 'META-INF/*.SF'
+    exclude 'META-INF/*.DSA'
+    exclude 'META-INF/*.RSA'
```

Comment:
i have no idea. It's probably because the spark jar is what gets used when running on a cluster and the cluster complains (it did't use to but it does now)